schema/schemas.openxmlformats.org/spreadsheetml/CT_ConditionalFormatting.go

```diff
@@ -17,7 +17,7 @@ import (
 type CT_ConditionalFormatting struct {
 	// PivotTable Conditional Formatting
 	PivotAttr *bool
-	// Sequence of Refernces
+	// Sequence of References
 	SqrefAttr *ST_Sqref
```

Comment:
Everything under schema is generated code, but don't worry about changing the commit.  I'll accept this PR and have already fixed the typo in the source it was generated from.